bms_test_.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import stream_bms_data
+
+class test_bms_stream(unittest.TestCase):

```

Comment:
The output format is the crucial functionality of the sender. Variations in that can throw the receiver out of gear. If future maintainers introduce an issue in the output format, it will be detected only during integration. That would take a lot more time than a simple unit-test.
Always test your crucial functionality.
In the code, it is clear that a `json.dumps` has been done to output the values. A test will safeguard that:
- no other text is put on the console - e.g., that some debugging prints are not left over in the delivery
- the order of printing is not guaranteed (so receivers should not assume an order)